src/content/docs/en/guides/integrations-guide/node.mdx

```diff
@@ -84,25 +84,26 @@ export default defineConfig({
 
 @astrojs/node can be configured by passing options into the adapter function. The following options are available:
 
-### Mode
+### `mode`
+<p>
+**Type:** `'middleware' | 'standalone'` <br />
+</p>
```

Comment:
```suggestion
**Default:** 'middleware' </br>
</p>
```
or undefined? and required?

Reply:
Good question! I'm not sure what the default is. I'll check.

Reply:
PL, there is no default. It's a required option.